cypress/support/commands.js

```diff
@@ -1,42 +1,42 @@
-import "cypress-localstorage-commands";
-import "cypress-wait-until";
-
-Cypress.Commands.add(
-  "setupSliceMachineUserContext",
-  ({
-    hasSendAReview, // boolean
-    isOnboarded, // boolean
-    updatesViewed, // object
-    hasSeenTutorialsTooTip, // boolean
-  }) => {
-    return cy.setLocalStorage(
-      "persist:root",
-      JSON.stringify({
-        userContext: JSON.stringify({
-          hasSendAReview,
-          isOnboarded,
-          updatesViewed: {
-            latest: null,
-            latestNonBreaking: null,
-            ...updatesViewed,
-          },
-          hasSeenTutorialsTooTip,
-        }),
-      })
-    );
-  }
-);
+import {
+  setSliceMachineUserContext,
+  cleanSliceMachineUserContext,
+  getSliceMachineUserContext,
+} from "../helpers/localStorage";
+import {
+  clearProject,
+  clearAssets,
+  clearCustomTypes,
+  clearSlices,
+  removeTypes,
+} from "../helpers/filesystem";
+import {
+  createCustomType,
+  renameCustomType,
+  addFieldToCustomType,
+} from "../helpers/customTypes";
+import { createSlice, renameSlice } from "../helpers/slices";
 
+/* -- LOCAL STORAGE -- */
+Cypress.Commands.add("setSliceMachineUserContext", setSliceMachineUserContext);
 Cypress.Commands.add(
   "cleanSliceMachineUserContext",
-  (hasSendAReview = true, isOnboarded = true) => {
-    return cy.removeLocalStorage("persist:root");
-  }
+  cleanSliceMachineUserContext
 );
+Cypress.Commands.add("getSliceMachineUserContext", getSliceMachineUserContext);
+
+/* -- PROJECT RESET -- */
+Cypress.Commands.add("clearProject", clearProject);
```

Comment:
Nice abstractions, it will really help the test readability 👍
Too bad Cypress doesn't offer a way to scope commands to organise them in a better way.